spring-pulsar/src/test/java/org/springframework/pulsar/core/PulsarTemplateTests.java

```diff
@@ -306,15 +308,15 @@ void sendNullWithDefaultTopicFails() {
 			PulsarProducerFactory<String> senderFactory = new DefaultPulsarProducerFactory<>(client,
 					"sendNullWithDefaultTopicFails");
 			PulsarTemplate<String> pulsarTemplate = new PulsarTemplate<>(senderFactory);
-			assertThatIllegalArgumentException().isThrownBy(() -> pulsarTemplate.send(null, Schema.STRING))
+			assertThatExceptionOfType(PulsarException.class).isThrownBy(() -> pulsarTemplate.send(null, Schema.STRING))
```

Comment:
With the suggested change, these change are not needed.

Reply:
Correct will be reverted